pkg/plugininventory/sqlite_inventory.go

```diff
@@ -458,6 +458,49 @@ func (b *SQLiteInventory) InsertPlugin(pluginInventoryEntry *PluginInventoryEntr
 	return nil
 }
 
+// InsertPluginGroup inserts plugin-group to the inventory
+// specifying override will delete the existing plugin-group and add new one
+func (b *SQLiteInventory) InsertPluginGroup(pg *PluginGroup, override bool) error {
```

Comment:
Yes, correct. The way we are using this will discard the DB if the insert/update queries fail. So, this should be fine. If Using SQLite transactions would be the better way if we were not discarding the DB.